invenio_app_ils/ui/src/invenio_app_ils/common/api/documents/serializer.js

```diff
@@ -34,6 +39,28 @@ function serializeResponse(hit) {
   return result;
 }
 
+function serializeDocuments(documents) {
```

Comment:
This serializer should be merged with the one above instead of creating new one - if needed. Let's discuss it.